### ISSUE TITLE
Uncaught TypeError: Cannot read properties of null (reading 'register') at emberCliEmbeddedStart (embedded.ts:112:14)

### ISSUE THREAD

**Issue by JamesWilmot** [peopledoc/ember-cli-embedded#186]
Trying to use addon with basically an empty ember 4.4 app and I'm getting this error:

```
Uncaught TypeError: Cannot read properties of null (reading 'register')
    at emberCliEmbeddedStart (embedded.ts:112:14)
```

I'm going to poke around and see what I can do. I'll pick my co-worker's brain when she gets back too.

**Comment by MrChocolatine** [comment]
Attempt to fix this issue with #188 .

**Comment by ndekeister-us** [comment]
@JamesWilmot released v2.5.0 with commit from @MrChocolatine which fix the issue, thank you both for report + fix!
